cmd/crc/cmd/status.go

```diff
@@ -38,10 +38,11 @@ var statusCmd = &cobra.Command{
 }
 
 type Status struct {
-	CrcStatus       string `json:"crcStatus"`
+	CrcStatus       string `json:"virtualMachineStatus"`
```

Comment:
This change don't belong to this commit.

Reply:
yes you're right. I was not sure of this change. WDYT of virtualMachineStatus ?

Reply:
i'd prefer it to be called `crcStatus` only, as this also relates to the struct member name, if that makes sense..

Reply:
I put back `crcStatus`